libraries/AP_Motors/AP_MotorsMulticopter.h

```diff
@@ -102,6 +102,8 @@ class AP_MotorsMulticopter : public AP_Motors {
     // 10hz logging of voltage scaling and max trust
     void                Log_Write() override;
 
+    bool                check_failsafe() override;
```

Comment:
it would be good to add a comment above this method

Reply:
Fixed